environment/src/test/java/com/sequenceiq/environment/credential/service/CredentialServiceTest.java

```diff
@@ -67,70 +91,93 @@
     @MockBean
     private CloudbreakMessagesService cloudbreakMessagesService;
 
+    @BeforeEach
+    public void setupTestCredential() {
+        CREDENTIAL.setName(CREDENTIAL_NAME);
+        CREDENTIAL.setCloudPlatform(PLATFORM);
+        String credentialAttributesSecret = getTestAttributes(STATE, DEPLOYMENT_ADDRESS, REDIRECT_URL);
+        CREDENTIAL.setAttributes(credentialAttributesSecret);
+    }
+
+    private String getTestAttributes(String state, String deploymentAddress, String url) {
```

Comment:
could you please move all the private - helper - methods to the bottom, to keep the calling order (top to down)?
thank you!

Reply:
ok